lib/docs.js

```diff
@@ -106,6 +106,7 @@
  * @param {number} [args.touchZoomMin=0.5] - The minimum zoom scale.
  * @param {number} [args.touchZoomMax=1.75] - The maximum zoom scale.
  * @param {number} [args.maxPixelRatio=1.5] - The maximum pixel ratio for high DPI displays.  High DPI displays can cause sluggish performance, this caps resolution the grid is rendered at.  Standard resolution (e.g.: 1920x1080) have a pixel ratio of 1:1 while higher resolution displays can be higher (e.g.: Retina display 2:1).  Higher numbers are sharper (higher resolution) up to the max of your display (usually 2), lower numbers are lower resolution, down to 1.  It might be fun to set a value lower than 1, I've never done it.
+ * @param {object} [args.fillCellCallback] - The callback function that is called when a cell is filled.
```

Comment:
I had to look up how to describe a function `@param` (is `Function` or `function`?), but then discovered you can describe [@callback types](https://jsdoc.app/tags-callback.html) and refer to them by name. For example:

```
@callback fillCellCallback
@param {object} args

// and then later:

@param {fillCellCallback} [args.fillCellCallback] - The callback function that is called when a cell is filled.
```

I don't _exactly_ know how that'll work within the context of `docs.js`, because it feels like all those params belong to the same thing, and I'm not sure what happens when you introduce a `@callback` and then some new `@param`s on subsequent lines — those _should_ belong to the callback, but any other params below shouldn't... no time to explore right now, so just leaving this as a note here. (You don't need to change anything, I'm just thinking out loud — I think a lot of this will be fixed if start introducing TypeScript annotations, like [@hangxingliu is doing in his pull request](https://github.com/TonyGermaneri/canvas-datagrid/pull/498), and using _those_ as basis for documentation)


Also, next step would be to describe the shape of `args` passed to the callback, because `{object}` isn't very descriptive, obviously.

Reply:
Looks great!